auth-web/src/components/auth/common/AccountSuspendAlert.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+<v-card flat class="" color="error">
+  <v-alert :icon="false" prominent type="error" class="mb-0">
+    <v-row>
+      <v-col cols="9">
+        <div class="banner-info">
+          <v-icon color="link" left x-large>mdi-information-outline</v-icon>
+          <div class="ml-4">
+            <div class="font-weight-bold">Account Suspended</div>
+            <div>Account has been suspended for outstanding balance (NSF).</div>
+          </div>
+        </div>
+      </v-col>
+      <v-col class="text-end">{{suspendedDate}}</v-col>
+    </v-row>
+    <v-row>
+      <v-col cols="9" class="font-weight-bold balance">BALANCE DUE:  ${{totalAmountToPay.toFixed(2)}}</v-col>
+    </v-row>
+  </v-alert>
+</v-card>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import CommonUtils from '@/util/common-util'
+import { FailedInvoice } from '@/models/invoice'
+import { Organization } from '@/models/Organization'
+import { namespace } from 'vuex-class'
+
+const OrgModule = namespace('org')
+
+@Component
+export default class AccountSuspendAlert extends Vue {
+  @OrgModule.Action('calculateFailedInvoices') private calculateFailedInvoices!: () => FailedInvoice
+  @OrgModule.State('currentOrganization') private currentOrganization!: Organization
+  private formatDate = CommonUtils.formatDisplayDate
+
+  private totalTransactionAmount: number = 0
```

Comment:
u dont need a explicit type if u assign it to 0..
 private totalTransactionAmount = 0 will be enough i think